### PR TITLE
WIP: Add Direct Byte Buffer native memory allocation to javacores

### PR DESCRIPTION
**Pull request by adamfarley**
Right now we need an extension to do this.

This change removes the need for that extension.

Signed-off-by: Adam Farley <adam.farley@uk.ibm.com>

### CODE REVIEW

File: debugtools/DDR_Artifacts/specs/29/aix_ppc [hunk 1]
```diff
@@ -7624,6 +7624,8 @@ C|J9VMCONSTANTPOOL_JAVALANGVIRTUALMACHINEERROR
 C|J9VMCONSTANTPOOL_JAVALANGVOID
 C|J9VMCONSTANTPOOL_JAVAUTILCONCURRENTLOCKSABSTRACTOWNABLESYNCHRONIZER
 C|J9VMCONSTANTPOOL_JAVAUTILCONCURRENTLOCKSABSTRACTOWNABLESYNCHRONIZER_EXCLUSIVEOWNERTHREAD
+C|J9VMCONSTANTPOOL_JAVANIOBITS_COUNT
```
Comment (pshipton) [inline debugtools/DDR_Artifacts/specs/29/aix_ppc]: You shouldn't need to modify any of the DDR specs or the superset.

File: runtime/oti/vmconstantpool.xml [hunk 2]
```diff
@@ -358,6 +358,10 @@
 	<!-- Security manager check -->
 	<staticfieldref class="java/lang/System" name="security" signature="Ljava/lang/SecurityManager;"/>
 
+	<!-- Access to Direct Byte Buffer native memory allocation data -->
+	<staticfieldref class="java/nio/Bits" name="count" signature="Ljava/util/concurrent/atomic/AtomicLong;"/>
+	<staticfieldref class="java/nio/Bits" name="reservedMemory" signature="Ljava/util/concurrent/atomic/AtomicLong;"/>
+
```
Comment (pshipton) [inline runtime/oti/vmconstantpool.xml]: You'll also need an entry for the `value` field in AtomicLong.

Comment (adamfarley) [reply runtime/oti/vmconstantpool.xml]: Done. It was faster to make a new PR.

New PR: https://github.com/eclipse/openj9/pull/1508

File: debugtools/DDR_Artifacts/specs/29/aix_ppc (hunk 1 above)
Comment (adamfarley) [reply debugtools/DDR_Artifacts/specs/29/aix_ppc]: Done. It was faster to make a new PR.

New PR: https://github.com/eclipse/openj9/pull/1508

**Pull request closed without merging**